wallet/test/Test/Pos/Wallet/Web/Util.hs

```diff
@@ -206,5 +205,5 @@ newtype DerivingIndex = DerivingIndex
     } deriving (Eq, Num, Ord)
 
 instance Arbitrary DerivingIndex where
-    arbitrary = DerivingIndex <$> choose (0, firstHardened - 1)
+    arbitrary = DerivingIndex <$> choose (firstHardened, firstHardened + (firstHardened - 1))
```

Comment:
> `firstHardened + (firstHardened - 1)`

It looks ungood (  You effectively bound to specific value of `firstHardened`. 
Why not just `maxBound`?